ja/models/associations-linking-models-together.rst

```diff
@@ -356,14 +345,111 @@ belongsToアソシエーションの配列に指定できるキーは以下の
             )
     )
 
+counterCache - count() 結果をキャッシュする
+===========================================
+
+この機能は、関連データの件数をキャッシュしてくれます。
+``find('count')`` でデータ件数を取得する代わりに、
+``$hasMany`` アソシエーションのモデルがデータの追加及び削除を追跡して、
+データ件数を示すフィールドを増減してくれます。
+
+フィールドの名前は以下のように、モデルの単数形にアンダースコアで
+"count" をつなげます。 ::
+
+    my_model_count
+
+``ImageComment`` と ``Image`` というモデルを準備して、 ``images`` テーブルに
+``image_comment_count`` という名前の INT フィールドを追加しましょう。
+
+以下のサンプルを参考にしてください。
+
+========== ======================= =========================================
+モデル     アソシエーションモデル  サンプル
```

Comment:
The width of Japanese character depends on font.
GitHub use proportional font.  For rendering correctly, use monospaced font .